index.html

```diff
@@ -91,13 +148,14 @@ <h2 style="color:#ffff00;">Organ</h2>
             <p>Structure made up of a group of tissues, working together to perform specific functions</p>
             <h2 style="color:#ffff00;">Organ System</h2>
             <p>Group of organs with related functions, working together to perform body functions</p>
-    
-        </div>
-    </section>
+        </section>
+    </div>
 
     <!-- End Site Content -->
     <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
     <script>window.jQuery || document.write('<script src="scripts/lib/jquery-1.9.1.min.js"><\/script>')</script>
```

Comment:
I know this was already here, but if we have a local version we should use that instead of relying and checking on the status of a CDN.